### ISSUE TITLE
High-contrast mode should have even stronger contrast in dark mode

### ISSUE THREAD

**Issue by Joshix-1** [mastodon/mastodon#39693]
### Steps to reproduce the problem

open mastodon interface in dark mode

### Expected behaviour

Contrast should be better and visibly increased by highcontrast mode

### Actual behaviour

high contrast mode only visibly increases border contrast and not text contrast

### Detailed description

In white mode #fff is a high contrast background (too bright for my taste, but I don't use lightmode). It has fine contrast.

The gray backrgound in the dark mode is too bright for my taste (or the text too dark or both).

### Mastodon instance

mastodon.social

### Mastodon version

v4.6.2

### Browser name and version

Firefox 162

### Operating system

Arch Linux

**Comment by diondiondion** [comment]
Hi @Joshix-1, the text contrast in 4.6's high-contrast mode isn't that different from the default theme because the default theme already has pretty good contrast now. Compared to Mastodon 4.5, both the default and high-contrast modes have better/more consistently high contrast and meet the WCAG AA minimum contrast requirements.

We're planning to allow for more user control over theming in the near future, but until then I've moved this issue from "Bug" to "Suggestion".

**Comment by Joshix-1** [comment]
The default light theme has a much higher contrast than the default dark theme.

**Comment by diondiondion** [comment]
That's not unusual, white is a very common background colour while pure black isn't.
